---
layout: default
---

<h3> A workshop at the Annual Conference on Neural Information Processing Systems.<br /> Saturday Dec 9th 2017, Long Beach Convention Center, Long Beach, CA, USA</h3>

## Important dates
   * Submissions due: ~~**Oct 13th, 2017, 11:59pm (time zone of your choice)**~~ **Extended: Oct 15th, 2017, 11:59pm (time zone of your choice)** [MLCB submission web site](https://easychair.org/conferences/?conf=nipsmlcb2017)
   * Decision notification: ~~Oct 30th, 2017~~
   * Workshop: Dec 9th, 2017


## Instructions for posters
The posters dimensions are 36 x 48 in. (91cm x 122cm). **Please note**: this is different than previous years.

## Invited Speakers
   * [Eran Halperin - UCLA (USA)](https://www.eranhalperingenomics.com//)
   * [Christina Leslie - Memorial Sloan Kettering Cancer Center (USA)](https://www.mskcc.org/research-areas/labs/christina-leslie)
   * [Ben Raphael - Princeton (USA)](http://compbio.cs.brown.edu/)

## Organizers
   * [Nicolo Fusi - Microsoft Research (USA)](http://www.nicolofusi.com)
   * [Sara Mostafavi - University of British Columbia (Canada)](http://saramostafavi.github.io/)
   * [Anshul Kundaje - Stanford (USA)](https://sites.google.com/site/anshulkundaje/)
   * [Gerald Quon - UC Davis (USA)](http://qlab.faculty.ucdavis.edu/)
   * [James Zou - Stanford (USA)](https://sites.google.com/site/jamesyzou/)

## Schedule
```
8:55 – 9:00     Opening remarks

9:00 – 9:40     Invited Speaker 1 – Christina Leslie

9:40 – 9:55     Talk 1 -- Denoising scRNA-seq Data Using Deep Count Autoencoders (Gökcen Eraslan)
9:55 – 10:10    Talk 2 -- Fine Mapping of Chromatin Interactions via Neural Nets (Artur Jaroszewicz)
10:10 – 10:25   Talk 3 -- TF-MoDISco: Learning High-Quality, Non-Redundant Transcription Factor Binding Motifs Using Deep Learning (Avanti Shrikumar)

10:30 – 11:00   Coffee break + posters

11:00 –12:00    Spotlights

12:00 – 1:30    Lunch + Posters

1:30 – 2:00     Posters

2:00 – 2:40     Invited Speaker 2 – Eran Halperin

2:40 – 2:55     Talk 4 – Variational Bayes inference algorithm for causal multivariate mediation with linkage disequilibrium (Yongjin Park)
2:55 – 3:10     Talk 5 – Reference-Free Archaic Admixture Segmentation Using A Permutation-Equivariant Network (Jeffrey Chan)
3:10 – 3:25     Talk 6 -- Robust and Scalable Models of Microbiome Dynamics for Bacteriotherapy Design (Travis Gibson)
           
3:30 – 4:00     Coffee break + posters

4:00 – 4:40     Invited Speaker 3 – Ben Raphael

4:40 – 4:55     Talk 7 -- Drug Response Variational Autoencoder (Ladislav Rampášek)
4:55 – 5:10     Talk 8 -- Variational auto-encoding of protein sequences (Sam Sinai)
5:10 – 5:25     Talk 9 -- Antigen Identification for Cancer Immunotherapy by Deep Learning on Tumor HLA Peptides 

5:25 – 5:35     Sponsors + prizes

```

## Description
The field of computational biology has seen dramatic growth over the past few years. A wide range of high-throughput technologies developed in the last decade now enable us to measure parts of a biological system at various resolutions—at the genome, epigenome, transcriptome, and proteome levels. These technologies are now being used to collect data for an ever-increasingly diverse set of problems, ranging from classical problems such as predicting differentially regulated genes between time points and predicting subcellular localization of RNA and proteins, to models that explore complex mechanistic hypotheses bridging the gap between genetics and disease, population genetics and transcriptional regulation. Fully realizing the scientific and clinical potential of these data requires developing novel supervised and unsupervised learning methods that are scalable, can accommodate heterogeneity, are robust to systematic noise and confounding factors, and provide mechanistic insights. 

The goals of this workshop are to i) present emerging problems and innovative machine learning techniques in computational biology, and ii) generate discussion on how to best model the intricacies of biological data and synthesize and interpret results in light of the current work in the field. We will invite several leaders at the intersection of computational biology and machine learning who will present current research problems in computational biology and lead these discussions based on their own research and experiences. We will also have the usual rigorous screening of contributed talks on novel learning approaches in computational biology. We encourage contributions describing either progress on new bioinformatics problems or work on established problems using methods that are substantially different from established alternatives. Deep learning, kernel methods, graphical models, feature selection, non-parametric models and other techniques applied to relevant bioinformatics problems would all be appropriate for the workshop. We will also encourage contributions to address new challenges in analyzing data generated from gene editing, single cell genomics and other novel technologies. The targeted audience are people with interest in machine learning and applications to relevant problems from the life sciences, including NIPS participants without any existing research link to computational biology. Many of the talks will be of interest to the broad machine learning community.  


## Sponsors
<a href='https://recursionpharma.com/careers/'><img src='http://mlcb.github.io/recursion-1A.png' width='60%' height='60%'></a>
<a href='http://www.amazon.com/'><img src='http://mlcb.github.io/amazon_logo_RGB.jpg' width='70%' height='70%'></a>
<a href='https://www.deepgenomics.com/'><img src='http://mlcb.github.io/deep_genomics_logo_web.jpg' width='80%' height='80%'></a>

## Program Committee

Jean-Philippe	Vert	(Ecole des Mines de Paris)  
Karen	Sachs	(Stanford University)   
Li	Shen	(Icahn School of Medicine at Mount Sinai)     
Benjamin	(Haibe-Kains)   
Yun	Song	(University of California, Berkeley)   
Laurent	Jacob	(Mines ParisTech)   
Jian	Ma	(Carnegie Mellon University)   
Yves	Moreau	(Katholieke Universiteit Leuven)   
Andrew	Delong	(University of Toronto)   
Sushmita	Roy	(University of Wisconsin-Madison)   
Jinbo	Xu	Toyota (Technological Institute at Chicago)   
Christina	Leslie	(Memorial Sloan-Kettering Cancer Center)   
Nico	Pfeifer	(University of Tübingen)   
Karsten	Borgwardt	(ETH Zurich)  
Jason	Ernst	(University of California, Los Angeles)   
Maxwell	Libbrecht	(University of Washington Genome Sciences)   
Recep	Colak	(AWS)   
Juho	Rousu	(Aalto University)   
William Stafford	Noble	(University of Washington)   
Leopold	Parts	(Wellcome Trust Sanger Institute)   
Mathieu	Blanchette	(McGill University)  
Alexander	Schliep	(Gothenburg University)   
Claassen	Manfred	(ETH Zurich)   
Martin Renqiang	Min	(NEC Laboratories America)   
Antti	Honkela	(University of Helsinki)   
Maria	Chikina	(Mount Sinai School of Medicine)   
Oliver	Stegle	(EMBL-European Bioinformatics Institute)   
Cedric	Chauve	(Simon Fraser University)   
Gunnar	Ratsch	(Memorial Sloan Kettering Center)   
Michael M.	Hoffman	(Princess Margaret Cancer Centre/University of Toronto)   
Casey	Greene	(University of Pennsylvania)   
Anna	Goldenberg	(University of Toronto)   
Simon	Rogers	(Department of Computing Science, University of Glasgow)   
Pierre	Geurts	(University of Liège)   
Quaid	Morris	(University of Toronto)   
Alexis	Battle	(Johns Hopkins University)   
Chao	Cheng	(Dartmouth Medical School)   
Guido	Sanguinetti	(The University of Edinburgh)   
Bernard	Ng	(The University of British Columbia)   
Michael A.	Beer	(Johns Hopkins University)   
Su-In	Lee	(University of Washington)   
Anthony	Gitter	(University of Wisconsin-Madison)    

<!--## Submission instructions

Researchers interested in contributing should upload an extended abstract of 4 pages (excluding references)    in PDF format to the [MLCB submission web site](https://easychair.org/conferences/?conf=nipsmlcb2017) by Oct 13th, 2017, 11:59pm (time zone of your choice). Submissions exceeding the page limit will be automatically rejected.

No special style is required. Authors may use the NIPS style file, but are also free to use other styles as long as they use standard font size (11 pt) and margins (1 in). 

**Submissions should be suitably anonymized and meet the requirements for double-blind reviewing.**

All submissions will be anonymously peer reviewed and will be evaluated on the basis of their technical content.  A strong submission to the workshop typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online [here](http://raetschlab.org:10080/nipscompbio/).

The workshop allows submissions of papers that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at time of submission. 

Send any questions to nipsmlcb2017@easychair.org.-->


