---
layout: default
---

<h3> A workshop at the Annual Conference on Neural Information Processing Systems.<br /> Saturday Dec 10th 2016, Centre Convencions Internacional Barcelona, Barcelona, Spain </h3>

## Important dates
   * ~~Submissions due: **Oct 9, 2016, 11:59pm (time zone of your choice)** [MLCB submission web site](https://easychair.org/conferences/?conf=mlcb2016)~~
   * ~~Decision notification: Oct 21, 2016~~
   * Workshop: Dec 10, 2016

## Instructions for posters
The posters dimensions are 36 x 48 in. (91cm x 122cm). **Please note**: this is different than previous years.

## Invited Speakers
   * [Jonathan Marchini - University of Oxford (UK)](http://www.stats.ox.ac.uk/~marchini/)
   * [Alexis Battle - Johns Hopkins University (USA)](http://battlelab.jhu.edu/)

## Organizers
   * [Barbara Engelhardt - Princeton (USA)](https://www.cs.princeton.edu/~bee/)
   * [Nicolo Fusi - Microsoft Research (USA)](http://www.nicolofusi.com)
   * [Sara Mostafavi - University of British Columbia (Canada)](http://saramostafavi.github.io/)
   * [Gerald Quon - UC Davis (USA)](http://qlab.faculty.ucdavis.edu/)
   * [Oliver Stegle - EMBL-EBI (UK)](http://www.ebi.ac.uk/research/stegle)
   * [James Zou - Stanford (USA)](http://people.fas.harvard.edu/~jzou/)

## Schedule

Morning session I (9.00 - 10:00)  
08:35-08:40      Introduction  
08:40-09:25      _Keynote_ -  Jonathan Marchini  
09:25-09:45      _Contributed talk_ - Leo Jussi. Multiple Output Regression with Latent Noise.  
09:45-10:05      _Contributed talk_ - Sheng Wang. Predicting Protein Folding by Ultra-Deep Learning.  
10:05-10:25      _Contributed talk_ - Abhishek Sarkar. Dissecting the non-infinitesimal architecture of complex traits using group spike-and-slab priors.  

Morning coffee break (10.30 - 11.00)

11:00-12:30    Posters
12:30-13:30    Lunch
13:30-14:15    Keynote - Alexis Battle
14:15-14:35    _Contributed talk_ - Paul Blomstedt. Modelling-based experiment retrieval: A case study with gene expression clustering.
14:35-14:55    _Contributed talk_ - Alyssa Morrow. Convolutional Kitchen Sinks for Transcription Factor Binding Site Prediction.

Afternoon coffee break (15.00 - 15.30)

Afternoon session I (15.30 - 18.00)
15:30-15:50       _Contributed talk_ - Damien Arnol.  Modelling cell-cell interactions with spatial Gaussian processes.
15:50-16:10       _Contributed talk_ - Jennifer Listgarten. Predicting off-target effects for CRISPR guide design.
16:10-16:30       _Contributed talk_ - Aaron Schein. Beta Tucker decomposition for DNA methylation data.
16:30-16:50       _Contributed talk_ - Victoria Dean.. Deep Learning for Branch Point Selection in RNA Splicing.
16:50-17:10       _Contributed talk_ - Jane Hung. Applying Faster R-CNN for Object Detection on Malaria Images.
17:10-17:55       Panel Discussion - Deep learning and new technologies in compbio.  
17:55-18:00       Conclusion / goodbye / thank you


## Description
The field of computational biology has seen dramatic growth over the past few years. A wide range of high-throughput technologies developed in the last decade now enable us to measure parts of a biological system at various resolutionsâ€”at the genome, epigenome, transcriptome, and proteome levels. These technologies are now being used to collect data for an ever-increasingly diverse set of problems, ranging from classical problems such as predicting differentially regulated genes between time points and predicting subcellular localization of RNA and proteins, to models that explore complex mechanistic hypotheses bridging the gap between genetics and disease, population genetics and transcriptional regulation. Fully realizing the scientific and clinical potential of these data requires developing novel supervised and unsupervised learning methods that are scalable, can accommodate heterogeneity, are robust to systematic noise and confounding factors, and provide mechanistic insights. 

The goals of this workshop are to i) present emerging problems and innovative machine learning techniques in computational biology, and ii) generate discussion on how to best model the intricacies of biological data and synthesize and interpret results in light of the current work in the field. We will invite several leaders at the intersection of computational biology and machine learning who will present current research problems in computational biology and lead these discussions based on their own research and experiences. We will also have the usual rigorous screening of contributed talks on novel learning approaches in computational biology. We encourage contributions describing either progress on new bioinformatics problems or work on established problems using methods that are substantially different from established alternatives. Deep learning, kernel methods, graphical models, feature selection, non-parametric models and other techniques applied to relevant bioinformatics problems would all be appropriate for the workshop. We will also encourage contributions to address new challenges in analyzing data generated from gene editing, single cell genomics and other novel technologies. The targeted audience are people with interest in machine learning and applications to relevant problems from the life sciences, including NIPS participants without any existing research link to computational biology. Many of the talks will be of interest to the broad machine learning community.  


## Sponsors
 
[![Deep Genomics]({{ site.url }}/assets/dp_colored.png)](http://www.deepgenomics.com/)


## Submission instructions

Researchers interested in contributing should upload an extended abstract of 4 pages (excluding references) in PDF format to the [MLCB submission web site](https://easychair.org/conferences/?conf=mlcb2016) by Oct 9, 2016, 11:59pm (time zone of your choice). Submissions exceeding the page limit will be automatically rejected.

No special style is required. Authors may use the NIPS style file, but are also free to use other styles as long as they use standard font size (11 pt) and margins (1 in). 

**Submissions should be suitably anonymized and meet the requirements for double-blind reviewing.**

All submissions will be anonymously peer reviewed and will be evaluated on the basis of their technical content.  A strong submission to the workshop typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online [here](http://raetschlab.org:10080/nipscompbio/).

The workshop allows submissions of papers that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at time of submission. 

Send any questions to mlcb2016@easychair.org.


