---
layout: default
output:
  html_document:
    df_print: paged
---

<h3> Machine Learning in Computational Biology (MLCB) Meeting, co-located with NeurIPS. 
<br />  Dec 13th and 14th 2019, Vancouver, Canada</h3>

## About
We are excited to be holding the 14th MLCB meeting, co-located with NeurIPS in Vancouver. Since its inception in 2004, and until 2017, MLCB was an official NeurIPS workshop. Given the growth and maturity of the field, this year MLCB will be an independent conference co-located with NeurIPS. 

## Conference Location and Format
  * The conference will be located downtown Vancouver (a few minute walks from the Convention Center-NeurIPS location) at [UBC Robson Square](https://robsonsquare.ubc.ca/). 
  * We will have two full research days consisting of invited presentations and poster sessions. The meeting will be held in C300 on 13th (Theatre room), and C150/180 on Dec 14th, both at  UBC Robson Square -- 800 Robson St, Vancouver, BC V6Z 3B7.

## Important dates
   * Submissions due: ~~**Sept 203th, 2019, 11:59pm**~~ **Extended: Oct 4th, 2019, 11:59pm[MLCB submission web site](https://easychair.org/conferences/?conf=mlcb2019)
   * Decision notification: Oct 21th, 2019
   * Workshop: Dec 13-14th, 2019


<!--## Instructions for posters
The maximum poster dimensions are 2.5 feet wide, 4 feet tall. **Please note**: this is different than previous years.-->


<!--
All submissions will be anonymously peer reviewed and will be evaluated on the basis of their technical content.  A strong submission to the workshop typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online [here](http://raetschlab.org:10080/nipscompbio/).

The workshop allows submissions of papers that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at time of submission. 

Send any questions to nipsmlcb2017@easychair.org.-->




## Invited Speakers
   * [Quaid Morris - University of Toronto (Canada)](http://www.morrislab.ca/)
   * [Jennifer Listgarten - UC Berkeley (USA)](http://www.jennifer.listgarten.com/)
   * [William Stafford Noble - University of Washington (USA)](https://noble.gs.washington.edu/) 
   
   
## Organizers
   * [David Knowles - Columbia & New York Genome Center (USA)](https://davidaknowles.github.io/)
   * [Anshul Kundaje - Stanford (USA)](https://sites.google.com/site/anshulkundaje/)
   * [Su-In Lee - University of Washington (USA)](https://suinlee.cs.washington.edu/)
   * [Sara Mostafavi - University of British Columbia (Canada)](http://saramostafavi.github.io/)
   * [Gerald Quon - UC Davis (USA)](http://qlab.faculty.ucdavis.edu/)
   * [James Zou - Stanford (USA)](https://sites.google.com/site/jamesyzou/)


## Description
The field of computational biology has seen dramatic growth over the past few years. A wide range of high-throughput technologies developed in the last decade now enable us to measure parts of a biological system at various resolutions—at the genome, epigenome, transcriptome, and proteome levels. These technologies are now being used to collect data for an ever-increasingly diverse set of problems, ranging from classical problems such as predicting differentially regulated genes between time points and predicting subcellular localization of RNA and proteins, to models that explore complex mechanistic hypotheses bridging the gap between genetics and disease, population genetics and transcriptional regulation. Fully realizing the scientific and clinical potential of these data requires developing novel supervised and unsupervised learning methods that are scalable, can accommodate heterogeneity, are robust to systematic noise and confounding factors, and provide mechanistic insights. 

The goals of this workshop are to i) present emerging problems and innovative machine learning techniques in computational biology, and ii) generate discussion on how to best model the intricacies of biological data and synthesize and interpret results in light of the current work in the field. 

In addition to talks by invited speakers, will also have the usual rigorous screening of contributed talks on novel learning approaches in computational biology. The targeted audience are people with interest in machine learning and applications to relevant problems from the life sciences, including NIPS participants without any existing research link to computational biology. Many of the talks will be of interest to the broad machine learning community.  

## Submission instructions

Please see the Call for Paper (CfP) [here](https://docs.google.com/document/d/1IclEhfiiokboptsWqT0NhZeBL_WETb9IpVtd_PWsn4Y/edit?usp=sharing)


New this year we will accept submission in two tracks: 

1) Research track: abstracts describing research on novel learning approaches in computational biology. A strong submission to the workshop typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online at http://raetschlab.org:10080/nipscompbio/previous. We specially encourage submissions describing work in progress and early results, for generating discussions helpful in shaping the presented work.

2) Perspective track: position or review papers on important but controversial problems. Examples include (but are not limited to) different ways of conceptualizing the supposed dichotomy between interpretation and accuracy, over-interpretation from visual representations of high dimensional data, lack of statistical rigour in inferring conclusions from data and models. 

Researchers interested in contributing should upload an extended abstract of 4 pages (excluding references)    in PDF format to the [MLCB submission web site](https://easychair.org/conferences/?conf=mlcb2019) by Sep 20th, 2019, 11:59pm (PST). Submissions exceeding the page limit will be automatically rejected.

No special style is required. Authors may use the NIPS style file, but are also free to use other styles as long as they use standard font size (11 pt) and margins (1 in). 

**Submissions should be suitably anonymized and meet the requirements for double-blind reviewing.**


All submissions will be anonymously peer reviewed and will be evaluated on the basis of their technical content.  

The workshop allows submissions of papers (for both tracks) that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at the time of submission. 





## Schedule
```
TBA

```
## Program Committee

Elham	Azizi	(Columbia University/MSSM)<br>
Alexis	Battle	(Johns Hopkins University)<br>
Andreas	Beyer	(University of Cologne)<br>
Karsten	Borgwardt	(ETH Zurich)<br>
Chao	Cheng	(Dartmouth Medical School)<br>
Manfred	Claassen	(ETH Zurich)<br>
Andrew	Delong	(University of Toronto)<br>
Saso	Dzeroski	(Jozef Stefan Institute)<br>
Jason	Ernst	(University of California, Los Angeles)<br>
Pierre	Geurts	(University of Liège)<br>
Anthony	Gitter	(University of Wisconsin-Madison)<br>
Casey	Greene	(University of Pennsylvania)<br>
Michael M.	Hoffman	(Princess Margaret Cancer Centre/University of Toronto)<br>
Antti	Honkela	(University of Helsinki)<br>
Lin	Hou	(Tsinghua University)<br>
Laurent	Jacob	(Mines ParisTech)<br>
Smita	Krishnaswamy	(Yale University)<br>
Benjamin	Logsdon	(Sage Bionetworks)<br>
Jian	Ma	(Carnegie Mellon University)<br>
Martin Renqiang	Min	(NEC Laboratories America)<br>
Alan	Moses	(University of Toronto)<br>
Bernard	Ng	(The University of British Columbia)<br>
William Stafford	Noble	(University of Washington)<br>
Leopold	Parts	(Wellcome Trust Sanger Institute)<br>
Itsik	Pe'Er	(Columbia University)<br>
Nico	Pfeifer	(University of Tübingen)<br>
Yanjun	Qi	(University of Virginia)<br>
Magnus	Rattray	(The University of Manchester)<br>
Simon	Rogers	(University of Glasgow)<br>
Juho	Rousu	(Aalto University)<br>
Sushmita	Roy	(University of Wisconsin-Madison)<br>
Guido	Sanguinetti	(University of Edinburgh)<br>
Sriram	Sankararaman	(University of California, Los Angeles)<br>
Alexander	Schliep	(Gothenburg University)<br>
Li	Shen	(Icahn School of Medicine at Mount Sinai)<br>
David	van Dijk	(Yale University)<br>
Jean-Philippe	Vert	(Google Brain)<br>
Jinbo	Xu	(Toyota Technological Institute at Chicago)<br>
Marinka	Zitnik	(Stanford University/Harvard)<br>

<!--## 
## Sponsors
<a href='https://recursionpharma.com/careers/'><img src='http://mlcb.github.io/recursion-1A.png' width='60%' height='60%'></a>
<a href='http://www.amazon.com/'><img src='http://mlcb.github.io/amazon_logo_RGB.jpg' width='70%' height='70%'></a>


## Program Committee

Jean-Philippe	Vert	(Ecole des Mines de Paris)  
Karen	Sachs	(Stanford University)   
Li	Shen	(Icahn School of Medicine at Mount Sinai)     
Benjamin	(Haibe-Kains)   
Yun	Song	(University of California, Berkeley)   
Laurent	Jacob	(Mines ParisTech)   
Jian	Ma	(Carnegie Mellon University)   
Yves	Moreau	(Katholieke Universiteit Leuven)   
Andrew	Delong	(University of Toronto)   
Sushmita	Roy	(University of Wisconsin-Madison)   
Jinbo	Xu	Toyota (Technological Institute at Chicago)   
Christina	Leslie	(Memorial Sloan-Kettering Cancer Center)   
Nico	Pfeifer	(University of Tübingen)   
Karsten	Borgwardt	(ETH Zurich)  
Jason	Ernst	(University of California, Los Angeles)   
Maxwell	Libbrecht	(University of Washington Genome Sciences)   
Recep	Colak	(AWS)   
Juho	Rousu	(Aalto University)   
William Stafford	Noble	(University of Washington)   
Leopold	Parts	(Wellcome Trust Sanger Institute)   
Mathieu	Blanchette	(McGill University)  
Alexander	Schliep	(Gothenburg University)   
Claassen	Manfred	(ETH Zurich)   
Martin Renqiang	Min	(NEC Laboratories America)   
Antti	Honkela	(University of Helsinki)   
Maria	Chikina	(Mount Sinai School of Medicine)   
Oliver	Stegle	(EMBL-European Bioinformatics Institute)   
Cedric	Chauve	(Simon Fraser University)   
Gunnar	Ratsch	(Memorial Sloan Kettering Center)   
Michael M.	Hoffman	(Princess Margaret Cancer Centre/University of Toronto)   
Casey	Greene	(University of Pennsylvania)   
Anna	Goldenberg	(University of Toronto)   
Simon	Rogers	(Department of Computing Science, University of Glasgow)   
Pierre	Geurts	(University of Liège)   
Quaid	Morris	(University of Toronto)   
Alexis	Battle	(Johns Hopkins University)   
Chao	Cheng	(Dartmouth Medical School)   
Guido	Sanguinetti	(The University of Edinburgh)   
Bernard	Ng	(The University of British Columbia)   
Michael A.	Beer	(Johns Hopkins University)   
Su-In	Lee	(University of Washington)   
Anthony	Gitter	(University of Wisconsin-Madison)    
-->
<!--## Submission instructions

Researchers interested in contributing should upload an extended abstract of 4 pages (excluding references)    in PDF format to the [MLCB submission web site](https://easychair.org/conferences/?conf=nipsmlcb2017) by Oct 13th, 2017, 11:59pm (time zone of your choice). Submissions exceeding the page limit will be automatically rejected.

No special style is required. Authors may use the NIPS style file, but are also free to use other styles as long as they use standard font size (11 pt) and margins (1 in). 

**Submissions should be suitably anonymized and meet the requirements for double-blind reviewing.**

All submissions will be anonymously peer reviewed and will be evaluated on the basis of their technical content.  A strong submission to the workshop typically presents a new learning method that yields new biological insights, or applies an existing learning method to a new biological problem.  However, submissions that improve upon existing methods for solving previously studied problems will also be considered. Examples of research presented in previous years can be found online [here](http://raetschlab.org:10080/nipscompbio/).

The workshop allows submissions of papers that are under review or have been recently published in a conference or a journal. This is done to encourage presentation of mature research projects that are interesting to the community. The authors should clearly state any overlapping published work at time of submission. 

Send any questions to nipsmlcb2017@easychair.org.-->


