<!DOCTYPE HTML>
<head>
  <title>MLCB 2020 Proceedings</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="css/main.css" />
  <!--[if lte IE 8]><link rel="stylesheet" href="css/ie8.css" /><![endif]-->
</head>
<html>
  <body id="hello">
    <div class="container">
      <h2>MLCB 2020 Proceedings</h2>
      <p>The 15th Machine Learning in Computational Biology (MLCB) meeting was held virtually. For more information and recordings of the talks please see <a href='https://sites.google.com/cs.washington.edu/mlcb2020/'>here</a></p>
      <ul>
        <li>
          <i>Surag Nair, Avanti Shrikumar, Anshul Kundaje. </i>
          <b>fastISM: Performant in-silico saturation mutagenesis for convolutional neural networks. </b>
          <a href="papers/5_Nair_etal_2020.pdf">[pdf]</a>
          <a href="papers/5_Nair_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Maja Trebacz, Zohreh Shams, Mateja Jamnik, Paul Scherer, Nikola Simidjievski, Helena Andres Terre, Pietro Liò. </i>
          <b>Using ontology embeddings for structural inductive bias in gene expression data analysis. </b>
          <a href="papers/6_Trebacz_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Dimitrios Vitsios, Ryan S. Dhindsa, Ayal B. Gussow, Slavé Petrovski. </i>
          <b>Prioritization of non-coding variants based on human intolerance to variation and primary sequence context using deep learning. </b>
          <a href="papers/9_Vitsios_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Hannah Zhou, Avanti Shrikumar, Anshul Kundaje. </i>
          <b>Benchmarking Reverse-Complement Strategies for Deep Learning Models in Genomics. </b>
          <a href="papers/10_Zhou_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Paul Scherer, Maja Trebacz, Nikola Simidjievski, Zohreh Shams, Helena Andres Terre, Pietro Liò, Mateja Jamnik. </i>
          <b>Incorporating network based protein complex discovery into automated model construction. </b>
          <a href="papers/11_Scherer_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Jacob Schreiber, Yang Young Lu, William Stafford Noble . </i>
          <b>Ledidi: Designing genomic edits that induce functional activity. </b>
          <a href="papers/12_Schreiber_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Rui Yang, Alizera Karbalayghareh, Christina Leslie. </i>
          <b>HiC2Self: Self-supervised Hi-C contact map denoising. </b>
          <a href="papers/13_Yang_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Nicholas Nuechterlein, Beibin Li, Mehmet Saygin Seyfioglu, Sachin Mehta, Patrick J Cimino, Linda Shapiro. </i>
          <b>Leveraging Unlabeled Data for Glioma Molecular Subtype and Survival Prediction. </b>
          <a href="papers/15_Nuechterlein_etal_2020.pdf">[pdf]</a>
          <a href="papers/15_Nuechterlein_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Lotfi Slim, Clement Chatelain, Chloe-Agathe Azencott. </i>
          <b>Nonlinear post-selection inference for genome-wide association studies. </b>
          <a href="papers/16_Slim_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Joao Belo Pereira, Diogo Bastos, Erik Stroes, Evgeni Levin. </i>
          <b>Covered Information Disentanglement: Correcting Permutation Feature Importance in the Presence of Covariates. </b>
          <a href="papers/19_Pereira_etal_2020.pdf">[pdf]</a>
          <a href="papers/19_Pereira_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Hussein Mohsen, Jonathan Warrell, Martin Renqiang Min, Sahand Negahban, Mark Gerstein. </i>
          <b>Weight-based Neural Network Interpretability using Activation Tuning and Personalized Products. </b>
          <a href="papers/21_Mohsen_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Mukund Sudarshan, Susan E. Liao, Oded Regev. </i>
          <b>Novel Insights into RNA Splicing from Interpretable Machine Learning. </b>
          <a href="papers/22_Sudarshan_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Nancy-Sarah Yacovzada, Iris Kalka, Eran Segal, Eran Hornstein. </i>
          <b>A Method for Pedigree-based Propagated Risk Phenotyping on Clalit Data. </b>
          <a href="papers/24_Yacovzada_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Faezeh Bayat, Maxwell Libbrecht. </i>
          <b>VSS: Variance-stabilized signals for sequencing-based genomic signals. </b>
          <a href="papers/26_Bayat_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Morteza M. Saber, Maxwell Libbrecht, Leonid Chindelevitch, Jesse Shapiro. </i>
          <b>Do machine learning predictors of microbial phenotype from genotype identify causal variants?. </b>
          <a href="papers/28_Saber_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Ahmad Pesaranghader, Justin Pelletier, Jean-Christophe Grenier, Raphaël Poujol, Julie Hussin. </i>
          <b>ImputeCoVNet: 2D ResNet Autoencoder for Imputation of SARS-CoV-2 Sequences. </b>
          <a href="papers/29_Pesaranghader_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Johannes Linder, Alyssa La Fleur, Sreeram Kannan, Zibo Chen, Ajasja Ljubetic, David Baker, Georg Seelig. </i>
          <b>Efficient inference of nonlinear feature attributions with Scrambling Neural Networks. </b>
          <a href="papers/33_Linder_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Yun Feng, Christopher Yau. </i>
          <b>RLevolution: Unravelling the history of genomic instability through deep reinforcement learning. </b>
          <a href="papers/38_Feng_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Sergei Rybakov, Mohammad Lotfollahi, Fabian J. Theis, F. Alexander Wolf. </i>
          <b>Learning interpretable latent autoencoder representations with annotations of feature sets. </b>
          <a href="papers/40_Rybakov_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Andrew J. Jung, Leo J. Lee, Alice J. Gao, Brendan J. Frey. </i>
          <b>Machine Learning Approaches for RNA Editing Prediction. </b>
          <a href="papers/41_Jung_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Amy X. Lu, Alex X. Lu, Alan Moses. </i>
          <b>Evolution Is All You Need: Phylogenetic Augmentation for Contrastive Learning. </b>
          <a href="papers/44_Lu_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Amy X. Lu, Haoran Zhang, Marzyeh Ghassemi, Alan Moses . </i>
          <b>Self-Supervised Contrastive Learning of Protein Representations By Mutual Information Maximization. </b>
          <a href="papers/45_Lu_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Alireza Karbalaygharch, Christina Leslie. </i>
          <b>Modeling Gene Regulation by Integrating 1D and 3D Epigenomic Data with Graph Attention Networks. </b>
          <a href="papers/46_Karbalaygharch_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Dinesh Singh, Héctor Climente-González, Mathis Petrovich, Eiryo Kawakami, Makoto Yamada. </i>
          <b>FsNet: Feature Selection Network on High-dimensional Biological Data. </b>
          <a href="papers/54_Singh_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Zijun Zhang, Evan M. Cofer, Olga G. Troyanskaya. </i>
          <b>AMBIENT: Accelerated Convolutional Neural Network Architecture Search for Regulatory Genomics. </b>
          <a href="papers/55_Zhang_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Mohit Goyal, Guillermo Serrano, Ilan Shomorony, Mikel Hernaez, Idoia Ocha. </i>
          <b>JIND: Joint Integration and Neural Discrimination for automated single-cell annotation. </b>
          <a href="papers/57_Goyal_etal_2020.pdf">[pdf]</a>
          <a href="papers/57_Goyal_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Amr M. Alexandari, Connor Horton, Avanti Shrikumar, Polly Fordyce, Anshul Kundaje. </i>
          <b>Neural networks can extract thermodynamic DNA sequence affinities from in vivo binding profiles of transcription factor binding. </b>
          <a href="papers/58_Alexandari_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Aishwarya Mandyam, Yuhao Wan, Luis Ceze, Jeff Nivala, Kevin Jamieson. </i>
          <b>Molecular Matchmaker: selecting binding peptide-aptamer pairs using machine learning. </b>
          <a href="papers/59_Mandyam_etal_2020.pdf">[pdf]</a>
          <a href="papers/59_Mandyam_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Amir Shanehsazzadeh, David Belanger, David Dohan. </i>
          <b>Fixed-Length Protein Embeddings using Contextual Lenses. </b>
          <a href="papers/60_Shanehsazzadeh_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Einar Gabbasov, Miguel Moreno-Molina, I ̃naki Comas, Maxwell Libbrecht, Leonid Chin-delevitch. </i>
          <b>SplitStrains, a tool to identify and separate mixed Mycobacterium tuberculosis infections from WGS data. </b>
          <a href="papers/65_Gabbasov_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Neda Shokranch, Maxwell W Libbrecht. </i>
          <b>Graph embedding for inferring genome domains from genome 3D structure. </b>
          <a href="papers/66_Shokranch_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Sanjit Singh Batra, Jeffrey Spence, Yun S. Song. </i>
          <b>Learning putatively causal gene regulatory programs using permutation-equivariant neural networks. </b>
          <a href="papers/72_Batra_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Pinar Demetci, Rebecca Santorella, Bjorn Sandstede, William Stafford Noble,Ritambhara Singh. </i>
          <b>Gromov–Wasserstein based optimal transport to align single-cell multi-omics data. </b>
          <a href="papers/74_Demetci_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Antonio Moretti, Liyi Zhang, Itsik Pe’er. </i>
          <b>Variational Combinatorial Sequential Monte Carlo For Bayesian Phylogenetic Inference. </b>
          <a href="papers/75_Moretti_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Andres F. Duque, Guy Wolf, Kevin R. Moon. </i>
          <b>Visualizing High Dimensional and High Frequency Electrical Biosignals. </b>
          <a href="papers/76_Duque_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Anushri Arora, Tetsuhiro Harimoto, Tal Danino, Itsik Pe’er. </i>
          <b>Spatial Microbial Dynamics. </b>
          <a href="papers/79_Arora_etal_2020.pdf">[pdf]</a>
          <a href="papers/79_Arora_etal_2020_supp.pdf">[supplement]</a>
        </li>
        <li>
          <i>Manu Saraswat, Gherman Novakovsky. Etienne Meunier, Oriol Fomes, Sara Mostafavi, Wyeth W Wasserman. </i>
          <b>Convolutional Additive Models: a fully interpretable approach to Deep Learning in Genomics. </b>
          <a href="papers/84_Saraswat_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Omer Ronen, Or Zuk. </i>
          <b>Deep Homology-Based Protein Contact-Map Prediction. </b>
          <a href="papers/87_Ronen_etal_2020.pdf">[pdf]</a>
        </li>
        <li>
          <i>Habib Daneshpajouh, Bowen Chen, Neda Shokranch, Shohre Masoumi, Kay C Wiese, Maxwell W Libbrecht. </i>
          <b>Continuous chromatin state feature annotation of the human epigenome. </b>
          <a href="papers/89_Daneshpajouh_etal_2020.pdf">[pdf]</a>
        </li>
      </ul>
    </div>
  </body>
</html>