<!DOCTYPE HTML>
<head>
  <title>MLCB 2019 Proceedings</title>
  <meta charset="utf-8" />
  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <link rel="stylesheet" href="css/main.css" />
  <!--[if lte IE 8]><link rel="stylesheet" href="css/ie8.css" /><![endif]-->
</head>
<html>
  <body id="hello">
    <ul>
      <li>
        <i>Iddo Drori, Darshan Thaker, Arjun Srivatsa, Daniel Jeong, Yueqi Wang, Linyong Nan, Fan Wu, Dimitri Leggas, Jinhao Lei, Weiyi Lu, Weilong Fu, Yuan Gao, Sashank Karri, Anand Kannan, Antonio Moretti, Mohammed AlQuraishi, Chen Keasar and Itsik Pe'er. </i>
        <b>Accurate Protein Structure Prediction by Embeddings and Deep Learning Representations. </b>
        <a href="papers/paper_1.pdf">[pdf]</a>
      </li>
      <li>
        <i>Hyunmin Lee, Zhen Hao Wu, Carles Corbi-Verges, Mac Mok, Sidney Kang, Shun Liao, Zhaolei Zhang and Michael Garton. </i>
        <b>De Novo Crystallization Condition Prediction with Deep Learning. </b>
        <a href="papers/paper_3.pdf">[pdf]</a>
      </li>
      <li>
        <i>Frederick Matsen IV, Mathieu Fourment, Michael Karcher, Andrew Magee, Christiaan Swanepoel and Cheng Zhang. </i>
        <b>Learning, using, and extending variational distributions of phylogenetic trees. </b>
        <a href="papers/paper_5.pdf">[pdf]</a>
      </li>
      <li>
        <i>Sajal Kumar and Mingzhou Song. </i>
        <b>Statistical Inference of Discrete Combinatorial Functional Dependency in Biological Systems. </b>
        <a href="papers/paper_24.pdf">[pdf]</a>
      </li>
      <li>
        <i>David Yang, Samuel Goldman, Eli Weinstein and Debora Marks. </i>
        <b>Generative models for codon prediction and optimization. </b>
        <a href="papers/paper_29.pdf">[pdf]</a>
      </li>
      <li>
        <i>Michael Dimmick, Leo J Lee and Brendan J Frey. </i>
        <b>HiCSR: A Hi-C Super-Resolution Framework for Producing Highly Realistic Contact Maps. </b>
        <a href="papers/paper_30.pdf">[pdf]</a>
      </li>
      <li>
        <i>Pierre Boyeau, Romain Lopez, Jeffrey Regier, Adam Gayoso, Michael I. Jordan and Nir Yosef. </i>
        <b>Deep Generative Models for Detecting Differential Expression in Single Cells. </b>
        <a href="papers/paper_32.pdf">[pdf]</a>
      </li>
      <li>
        <i>Khawla Seddiki, Philippe Saudemont, Frederic Precioso, Nina Ogrinc, Maxence Wisztorski, Michel Salzet, Isabelle Fournier and Arnaud Droit. </i>
        <b>Feature learning with Deep Neural Networks for MS-based clinical diagnosis. </b>
        <a href="papers/paper_37.pdf">[pdf]</a>
      </li>
      <li>
        <i>Yiliang Zhang, Kexuan Liang, Molei Liu, Yue Li, Hao Ge and Hongyu Zhao. </i>
        <b>SCRIBE: a new approach to dropout imputation and batch effects correction for single-cell RNA-seq data. </b>
        <a href="papers/paper_39.pdf">[pdf]</a>
      </li>
      <li>
        <i>Alan Moses, Alex Lu, Amy Lu and Marzyeh Ghassemi. </i>
        <b>Transfer Learning vs. Batch Effects: what can we expect from neural networks in computational biology?. </b>
        <a href="papers/paper_41.pdf">[pdf]</a>
      </li>
      <li>
        <i>Adam Gayoso, Romain Lopez, Zoë Steier, Jeffrey Regier, Aaron Streets and Nir Yosef. </i>
        <b>A Joint Model of RNA Expression and Surface Protein Abundance in Single Cells. </b>
        <a href="papers/paper_43.pdf">[pdf]</a>
      </li>
      <li>
        <i>Lan Huong Nguyen and Susan Holmes. </i>
        <b>Diffusion t-SNE for multiscale data visualization. </b>
        <a href="papers/paper_45.pdf">[pdf]</a>
      </li>
      <li>
        <i>Hui Ting Grace Yeo and David Gifford. </i>
        <b>Disentangling unwanted sources of variation in single-cell RNA-sequencing data under weak supervision. </b>
        <a href="papers/paper_47.pdf">[pdf]</a>
      </li>
      <li>
        <i>Conor Delaney, Alexandra Schnell, Louis Cammarata, Aaron Yao-Smith, Aviv Regev, Vijay Kuchroo and Meromit Singer. </i>
        <b>COMET: A tool for predicting multiple gene-marker panels from single-cell transcriptomic data. </b>
        <a href="papers/paper_48.pdf">[pdf]</a>
      </li>
      <li>
        <i>Paul Bertin, Mohammad Hashir, Martin Weiss, Vincent Frappier, Theodore Perkins, Geneviève Boucher and Joseph Paul Cohen. </i>
        <b>Is graph-based feature selection of genes better than random?. </b>
        <a href="papers/paper_52.pdf">[pdf]</a>
      </li>
      <li>
        <i>Kevin Dsouza, Adam Li, Vijay Bhargava and Maxwell Libbrecht. </i>
        <b>A Cell Type-Agnostic Representation of the Human Epigenome through a Deep Recurrent Neural Network Model. </b>
        <a href="papers/paper_53.pdf">[pdf]</a>
      </li>
      <li>
        <i>Linhua Wang, Jeffrey Law, T. M. Murali and Gaurav Pandey. </i>
        <b>Data integration through heterogeneous ensembles for protein function prediction. </b>
        <a href="papers/paper_55.pdf">[pdf]</a>
      </li>
      <li>
        <i>Peter Koo and Matthew Ploenzke. </i>
        <b>Interpreting Deep Neural Networks Beyond Attribution Methods: Quantifying Global Importance of Features. </b>
        <a href="papers/paper_56.pdf">[pdf]</a>
      </li>
      <li>
        <i>Hassan Kané, Mohamed Coulibali, Ali Abdalla and Pelkins Ajanoh. </i>
        <b>Augmenting protein network embeddings with sequence information. </b>
        <a href="papers/paper_58.pdf">[pdf]</a>
      </li>
      <li>
        <i>Oscar Clivio, Romain Lopez, Jeffrey Regier, Adam Gayoso, Michael I. Jordan and Nir Yosef. </i>
        <b>Detecting Zero-Inflated Genes in Single-Cell Transcriptomics Data. </b>
        <a href="papers/paper_65.pdf">[pdf]</a>
      </li>
      <li>
        <i>Bernard Ng, William Casazza, Farnush Farhadi and Sara Mostafavi. </i>
        <b>Cascading Epigenomic Model for GWAS. </b>
        <a href="papers/paper_76.pdf">[pdf]</a>
      </li>
      <li>
        <i>Ammar Tareen and Justin Kinney. </i>
        <b>Biophysical models of cis-regulation as interpretable neural networks. </b>
        <a href="papers/paper_78.pdf">[pdf]</a>
      </li>
      <li>
        <i>Johannes Linder, Nicholas Bogard, Alexander B Rosenberg and Georg Seelig. </i>
        <b>Deep exploration networks for rapid engineering of functional DNA sequences. </b>
        <a href="papers/paper_87.pdf">[pdf]</a>
      </li>
      <li>
        <i>John Halloran and David Rocke. </i>
        <b>GPU-Accelerated SVM Learning for Extremely Fast Large-Scale Proteomics Analysis. </b>
        <a href="papers/paper_89.pdf">[pdf]</a>
      </li>
      <li>
        <i>Galip Gurkan Yardimci, Jacob Schreiber, Jeffrey Bilmes and William Stafford Noble. </i>
        <b>Selecting representative subsets of genomic loci. </b>
        <a href="papers/paper_92.pdf">[pdf]</a>
      </li>
      <li>
        <i>Shreshth Gandhi, Leo Lee, Andrew Delong, David Duvenaud and Brendan Frey. </i>
        <b>cDeepbind: A context sensitive deep learning model of RNA-protein binding. </b>
        <a href="papers/paper_97.pdf">[pdf]</a>
      </li>
      <li>
        <i>Camille Rochefort-Boulanger, Léo Choinière, Jean-Christophe Grenier, Pierre-Luc Carrier and Julie Hussin. </i>
        <b>Generalization Capability of the Diet Network Model on Genomic Data. </b>
        <a href="papers/paper_99.pdf">[pdf]</a>
      </li>
      <li>
        <i>Daniel Mas Montserrat, Carlos Bustamante and Alexander Ioannidis. </i>
        <b>Class-Conditional VAE-GAN for Local-Ancestry Simulation. </b>
        <a href="papers/paper_102.pdf">[pdf]</a>
      </li>
      <li>
        <i>Iman Deznabi, Büşra Arabacı, Mehmet Koyuturk and Oznur Tastan. </i>
        <b>DeepKinZero: Zero-Shot Learning for Predicting Kinase-Phosphosite Associations. </b>
        <a href="papers/paper_105.pdf">[pdf]</a>
      </li>
      <li>
        <i>Rohit Jammula, Vishnu Tejus and Shreya Shankar. </i>
        <b>Optimal Transfer Learning Model for Binary Classification of Funduscopic Images through Simple Heuristics. </b>
        <a href="papers/paper_107.pdf">[pdf]</a>
      </li>
      <li>
        <i>Jack Lanchantin and Yanjun Qi. </i>
        <b>Graph Convolutional Networks for Epigenetic State Prediction Using Both Sequence and 3D Genome Data. </b>
        <a href="papers/paper_110.pdf">[pdf]</a>
      </li>
      <li>
        <i>Sathappan Muthiah, Debanjan Datta, Mohammad Raihanul Islam, Patrick Butler, Andrew Warren and Naren Ramakrishnan. </i>
        <b>ProtTox: Toxin identification from Protein Sequences. </b>
        <a href="papers/paper_111.pdf">[pdf]</a>
      </li>
      <li>
        <i>Andreea Gane, David Belanger, David Dohan, Christof Angermueller, Ramya Deshpande, Suhani Vora, Olivier Chapelle, Babak Alipanahi, Kevin Murphy and Lucy Colwell. </i>
        <b>A Comparison of Generative Models for Sequence Design. </b>
        <a href="papers/paper_113.pdf">[pdf]</a>
      </li>
    </ul>
  </body>
</html>